libraries/cyclestreets-core/src/main/res/values-hi/arrays.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+  <string-array name="units">
+    <item>किलोमीटर</item>
+    <item>मील</item>
+  </string-array>
+  <string-array name="unit_values" translatable="false">
```

Comment:
As per the issue instructions, your PR should not include any items marked with the XML attribute `translatable="false"`.  Please remove these from all files in your submission.